dbms/src/test/java/org/polypheny/db/constraints/ForeignKeyConstraintTest.java

```diff
@@ -211,17 +186,18 @@ public void testInsertConflict() throws SQLException {
     }
 
 
-    @Test
-    @Ignore // todo dl enable as soon as such inserts work correctly
-    public void testInsertSelectNoConflict() throws SQLException {
+    @ParameterizedTest(name = "{index}. Create Index: {0}")
+    @ValueSource(booleans = { false, true })
+    @Disabled // todo dl enable as soon as such inserts work correctly
```

Comment:
Can you enable this test?

Reply:
Followed instructions, as they work now, removed.